### PR TITLE
Install rsyslog configuration file keeping Flocker logs out of /var/log

### PR DESCRIPTION
**Pull request by itamarst**
To review you need to install generated RPM somewhere and run the services. Things you will need to verify:

1. Flocker logs are not in /var/log/messages. Maybe... ssh into running acceptance test node?
2. Logs are still available via journald. You can validate this by checking that the remote logs for the CentOS acceptance builders still contains Flocker logs.

### CODE REVIEW

**Comment by jongiddy** [comment]
Packages for both CentOS and Ubuntu cannot be built due to missing ignores in `packaging.py`.

See https://github.com/ClusterHQ/flocker/blob/67d9f2aaab0ddcdb3a099d8d24422bfa2e054f5b/admin/packaging.py#L585 for an example of what is needed.

**Comment by itamarst** [comment]
Could not reproduce locally, so did a cargo-cult change. Hopefully this will fix it. Thanks for the exact pointer to what to do!

File: NEWS [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Flocker 1.0.2 (2015-06-XX)
 ==========================
 
+- On CentOS 7 Flocker logs are no longer written to /var/log/messages since this fills up disk space quickly and the logs are already available via journald. (FLOC-2534)
```
Comment (wallrj) [inline NEWS]: Needs a comma I think "On CentOS-7, Flocker logs are..."

Comment (wallrj) [inline NEWS]: Also maybe "since this fills up disk space quickly" > "to save disk space."

Comment (itamarst) [inline NEWS]: The former is more accurate :grin: This is serious "out of disk in a few hours" bug.

**Comment by wallrj** [comment]
I installed the packages from:
 * http://build.clusterhq.com/results/omnibus/release-maintenance/flocker-1.0.1/suppress-log-messages-FLOC-2541/centos-7/

On some centos-7 acceptance test nodes.

Verified that the new config files were present.

Restarted rsyslog

```
systemctl restart rsyslog
```

Rotated the logs

```
logrotate --force /etc/logrotate.d/syslog
```

Verified that the new log files no longer contain any flocker agent or control service logs

There are some warnings however:

```
Jun 30 17:26:11 acceptance-test-richardw-0 rsyslogd-2307: warning: ~ action is deprecated, consider using the 'stop' statement instead [try http://www.rsyslog.com/e/2307 ]
Jun 30 17:26:11 acceptance-test-richardw-0 rsyslogd-2307: warning: ~ action is deprecated, consider using the 'stop' statement instead [try http://www.rsyslog.com/e/2307 ]
Jun 30 17:26:11 acceptance-test-richardw-0 rsyslogd-2307: warning: ~ action is deprecated, consider using the 'stop' statement instead [try http://www.rsyslog.com/e/2307 ]
Jun 30 17:26:11 acceptance-test-richardw-0 rsyslogd-2307: warning: ~ action is deprecated, consider using the 'stop' statement instead [try http://www.rsyslog.com/e/2307 ]

```

I think we either need to figure out how to trigger a reconfiguration of the rsyslog daemon, or document that rsyslog needs restarting / reconfiguring after installing flocker.

Create a followup issue for that if you think it can be postponed for a later release.

**Pull request merged**